rpi-eeprom-config

```diff
@@ -22,18 +27,128 @@ MAGIC_MASK = 0xfffff00f
 FILE_MAGIC = 0x55aaf11f # id for modifiable file, currently only bootconf.txt
 FILE_HDR_LEN = 20
 FILENAME_LEN = 12
+TEMP_DIR = None
+
+def exit_handler():
+    """
+    Delete any temporary files.
+    """
+    if TEMP_DIR is not None and os.path.exists(TEMP_DIR):
+        tmp_image = '%s/pieeprom.upd' % TEMP_DIR
+        if os.path.exists(tmp_image):
+            os.remove(tmp_image)
+        tmp_conf = '%s/boot.conf' % TEMP_DIR
+        if os.path.exists(tmp_conf):
+            os.remove(tmp_conf)
+        os.rmdir(TEMP_DIR)
```

Comment:
Alternatively, you could just use [shutil.rmtree](https://docs.python.org/3/library/shutil.html#shutil.rmtree) and skip the file-exists tests :wink:

Reply:
That's deliberate. If the exists tests fails then it indicates a bug in a safer way than a random recursive directorty retre removal